### ISSUE TITLE
LDRS 620 Activity 1.2 H5P

### ISSUE THREAD

**Issue by karenroeck** [TWUOnline/bug-tracking#345]
**Which course contains the bug?**  
LDRS 620 FAR Centre MASTER
 
**Describe the error and/or your suggestion**  
Please create an H5P activity for Activity 1.2 where students can slide to choose their answers to each question: "Strongly Disagree | Disagree | Neutral | Agree | Strongly Agreee"
 
**Location of the bug**  
Please provide a specific link to the page with the error, as well as where the error is on the page:
1. URL:  https://learn.twu.ca/course/view.php?id=15710&section=1
2. Unit #1      Topic #1       Activity#1.2
 
**Optional: Screenshots**

**Comment by rezakh66** [comment]
@karenroeck Hi Karen,
I could not find any survey type activity in H5P but I found this kind of multiple-choice closest to that. Can you please check and let me know your thoughts?

**Comment by kmarjanovic** [comment]
@rezakh66 @karenroeck 
That could definitely work...although it would be nice to find a slider option.  I wonder if there are WordPress plugins that could work.
@cmadland or @Jeff-TWU do you know of any media plugins that have a slider option for a survey activity?

Since students are reflecting on their answers - at the end they'll need to see the results or a summary.  Which media option gives us this?

**Comment by jb0ob** [comment]
<img width="753" alt="Screen Shot 2020-07-17 at 11 11 40 AM" src="https://user-images.githubusercontent.com/39725337/87817734-56db2d00-c81e-11ea-8d65-538a7c681714.png">
Without being able to see things in context, I can't really respond properly. It appears I only have student access? @cmadland  are you able to give me access?

**Comment by Jeff-TWU** [comment]
<img width="753" alt="Screen Shot 2020-07-17 at 11 11 40 AM" src="https://user-images.githubusercontent.com/66963480/87817930-a9b4e480-c81e-11ea-9c9c-27f6edfba197.png">
I'm unable to properly see things in context. @cmadland would you be able to set me up with proper access?

**Comment by karenroeck** [comment]
@Jeff-TWU in the meantime, here's a screenshot of the survey. The top is the H5P multiple choice option, and underneath is the way it appears in Moodle. 

![Screen Shot 2020-07-17 at 11 36 20 AM](https://user-images.githubusercontent.com/67760065/87819885-da4a4d80-c821-11ea-9139-17651ca79832.png)

**Comment by cmadland** [comment]
Jeff, I've added you to that course, so you can see it in context.

We will have to figure out how much general access to Moodle you will need. For now, maybe one-off access is ok?

Colin
________________________________

**Comment by Jeff-TWU** [comment]
@cmadland Thanks Colin! One-offs sound good to me.

**Comment by Jeff-TWU** [comment]
So I did some research, there's a few options (many of them paid).

I like Questionaire as it's sticking within H5P and would be very easy to implement. Students would have the option of reviewing their answers (one by one) before submitting a survey which could then be sent out later.
https://h5p.org/questionnaire

Responsive Poll is interesting, but I'm not sure it would be capable of being a part of a slider. I think it's interesting that it shows results immediately, comparing all submitted results.

https://codecanyon.net/item/responsive-poll/6785692

I found some other paid quiz-like ones but I don't think their would be any review option, and they had a monthly subscription.
